FirebaseAuth/Sources/Swift/Utilities/AuthRecaptchaVerifier.swift

```diff
@@ -94,38 +98,29 @@
 
     func verify(forceRefresh: Bool, action: AuthRecaptchaAction) async throws -> String {
       try await retrieveRecaptchaConfig(forceRefresh: forceRefresh)
-      if recaptchaClient == nil {
-        guard let siteKey = siteKey(),
-              let recaptcha: AnyClass = NSClassFromString("Recaptcha") else {
-          throw AuthErrorUtils.recaptchaSDKNotLinkedError()
-        }
-        recaptchaClient = try await recaptcha.getClient(withSiteKey: siteKey)
-      }
-      return try await retrieveRecaptchaToken(withAction: action)
-    }
-
-    func retrieveRecaptchaToken(withAction action: AuthRecaptchaAction) async throws -> String {
-      guard let actionString = actionToStringMap[action],
-            let RecaptchaActionClass = NSClassFromString("RecaptchaAction"),
-            let actionClass = RecaptchaActionClass as? any RCAActionProtocol.Type else {
-        throw AuthErrorUtils.recaptchaSDKNotLinkedError()
+      guard let siteKey = siteKey() else {
+        throw AuthErrorUtils.recaptchaSiteKeyMissing()
       }
-      let customAction = actionClass.init(customAction: actionString)
-      do {
-        let token = try await recaptchaClient?.execute(withAction: customAction)
-        AuthLog.logInfo(code: "I-AUT000100", message: "reCAPTCHA token retrieval succeeded.")
-        guard let token else {
-          AuthLog.logInfo(
-            code: "I-AUT000101",
-            message: "reCAPTCHA token retrieval returned nil. NO_RECAPTCHA sent as the fake code."
-          )
-          return "NO_RECAPTCHA"
+      let actionString = actionToStringMap[action] ?? ""
+      return try await withCheckedThrowingContinuation { continuation in
+        FIRRecaptchaGetToken(siteKey, actionString,
+                             "NO_RECAPTCHA") { (token: String, error: Error?, linked: Bool) in
```

Comment:
The signature is:
```
void FIRRecaptchaGetToken(NSString *_Nonnull siteKey,
                          NSString *_Nonnull fakeToken,
                          NSString *_Nonnull actionString,
                          _Nonnull FIRAuthRecaptchaTokenCallback callback);
```
The second and third parameters look to be reversed at the callsite.